modules/infra/variables.tf

```diff
@@ -21,3 +21,19 @@ variable "persistence_ebs_volumes_size" {
   default = 1
   type = number
 }
+
+# Configure RDS
+variable "pg_user" {
+  default = "dockup_user"
+  type = string
+}
+
+variable "pg_password" {
+  default = "sup3rs3cr3tpAssw0rd"
```

Comment:
Lets not provide default value here or use this https://www.terraform.io/docs/providers/random/r/string.html